pkg/server/cloud/aws/instances_test.go

```diff
@@ -67,3 +67,51 @@ func TestBootImageSpecToDescribeImagesInput(t *testing.T) {
 		assert.Equal(t, tc.Input, *input)
 	}
 }
+
+func TestGetRootDeviceVolumeSize(t *testing.T) {
+	notRootDeviceName := "not-root"
+	rootDeviceName := "root-device"
+	var volumeSize int64 = 100
+	var expectedVolumeSize int32 = 100
+	testCases := []struct{
+		caseName string
+		blockDevices []*ec2.BlockDeviceMapping
+		rootDeviceName string
+		expectedRootDiskSize *int32
+	}{
+		{
+			"root-device-found",
+			[]*ec2.BlockDeviceMapping{
+				&ec2.BlockDeviceMapping{
+					DeviceName:  &notRootDeviceName,
+					Ebs:         nil,
+					NoDevice:    nil,
+					VirtualName: nil,
+				},
+				&ec2.BlockDeviceMapping{
+					DeviceName:  &rootDeviceName,
+					Ebs:         &ec2.EbsBlockDevice{
+						DeleteOnTermination: nil,
+						Encrypted:           nil,
+						Iops:                nil,
+						KmsKeyId:            nil,
+						SnapshotId:          nil,
+						VolumeSize:          &volumeSize,
+						VolumeType:          nil,
+					},
+					NoDevice:    nil,
+					VirtualName: nil,
```

Comment:
You can help readability here if leave out the pointers that are `nil`, since that's their default value.

Reply:
removed those nils ;)